usr/src/lib/libzfs_core/common/libzfs_core.c

```diff
@@ -548,8 +548,9 @@ recv_read(int fd, void *buf, int ilen)
 }
 
 static int
-lzc_receive_impl(const char *snapname, nvlist_t *props, const char *origin,
-    boolean_t force, boolean_t resumable, int fd)
+recv_impl(const char *snapname, nvlist_t *props, const char *origin,
```

Comment:
I renamed it just to avoid wrapping long lines elsewhere.
My thinking was that because this is a private function it does not have to have `lzc_` prefix.